go/pkg/sysdb/grpc/collection_service.go

```diff
@@ -54,8 +54,12 @@ func (s *Server) ResetState(context.Context, *emptypb.Empty) (*coordinatorpb.Res
 
 // The fact that we ignore the metadata of the generated collections is a
 // bit weird, but it is the easiest way to excercise all cases
+
+// NOTE: In current implementation, we do not support updating the metadata of an existing collection via this RPC.
```

Comment:
can we link to the PR from @itaismith that changed this behavior so future readers can benifet #2390